wnprc_billing/resources/web/wnprc_billing/form/field/ChargeItemField.js

```diff
@@ -122,8 +122,10 @@ function filterFunction(object,fieldBulk){
     }
 
     returnFilter.push(LABKEY.Filter.create('departmentCode', chargeGroup, LABKEY.Filter.Types.EQUAL));
-    returnFilter.push(LABKEY.Filter.create('startDate', dateValue.format("Y-m-d"), LABKEY.Filter.Types.DATE_LESS_THAN_OR_EQUAL));
-    returnFilter.push(LABKEY.Filter.create('endDate', dateValue.format("Y-m-d"), LABKEY.Filter.Types.DATE_GREATER_THAN_OR_EQUAL));
+    if (dateValue) {
+        returnFilter.push(LABKEY.Filter.create('startDate', dateValue.format("Y-m-d"), LABKEY.Filter.Types.DATE_LESS_THAN_OR_EQUAL));
```

Comment:
I know this is Daniel's code, but `.format()` isn't a method that's standard across all browsers. For example:

https://github.com/LabKey/onprcEHRModules/pull/337
```suggestion
        returnFilter.push(LABKEY.Filter.create('startDate', Ext4.Date.format(dateValue,"Y-m-d"), LABKEY.Filter.Types.DATE_LESS_THAN_OR_EQUAL));
```

Reply:
good eye, I did not pay attention to that detail, thanks!

Reply:
Using Ext4.Date.format() was actually the main fix in this issue - it loads Bulk Edit form as expected with an error message when dateValue is null:
![image](https://user-images.githubusercontent.com/10765266/233178613-eec77c32-f66e-4a91-ae33-25325b8f1ccd.png)

Adding the null check avoids the error "could not load records due to an unexpected error" seen in the screenshot above.